adapters/metax/metax.go

```diff
@@ -177,6 +178,17 @@ func getBidType(bid *openrtb2.Bid) (openrtb_ext.BidType, error) {
 	}
 }
 
+func getBidVideo(bid *openrtb2.Bid) *openrtb_ext.ExtBidPrebidVideo {
+	bidVideo := openrtb_ext.ExtBidPrebidVideo{}
+	if len(bid.Cat) > 0 {
+		bidVideo.PrimaryCategory = bid.Cat[0]
+	}
+	if bid.Dur > 0 {
+		bidVideo.Duration = int(bid.Dur)
+	}
```

Comment:
Can you please update one of your exemplary JSON tests so that the bid response contains a `cat` and `dur`? While the unit tests are nice they aren't required while coverage through JSON tests are.

Reply:
updated one JSON test